lib/core/src/sdk.rs

```diff
@@ -1728,17 +1753,21 @@ mod tests {
         ) => {{
             let swap = match $type {
                 "chain" => {
-                    let swap = new_chain_swap($direction.unwrap(), None);
+                    let swap = new_chain_swap(
+                        $args.direction.unwrap_or(Direction::Outgoing),
+                        $args.initial_payment_state,
+                        $args.accepts_zero_conf.unwrap_or(false),
```

Comment:
If `direction` and `accepts_zero_conf` default to these values here when not set, do they need to be optional at all? Why not make them non-optional and add a `impl Default for NewSwapArgs` with these 2 defaults?